docs/src/pages/guides/FHIRPerformanceGuide.md

```diff
@@ -181,11 +181,11 @@ TLS connection setup is a costly CPU operation. It is therefore important to ens
 
 ## 3.4. Value-Id Caches
 
-The IBM FHIR Server uses internal memory caches for resource type names, parameter names, references, codes and systems. These caches use a least-recently-used (LRU) strategy to avoid unbound growth which would result in an out-of-memory (OOM) condition.
+the LinuxForHealth FHIR Server uses internal memory caches for resource type names, parameter names, references, codes and systems. These caches use a least-recently-used (LRU) strategy to avoid unbound growth which would result in an out-of-memory (OOM) condition.
```

Comment:
```suggestion
The LinuxForHealth FHIR Server uses internal memory caches for resource type names, parameter names, references, codes and systems. These caches use a least-recently-used (LRU) strategy to avoid unbound growth which would result in an out-of-memory (OOM) condition.
```